managed-databases/postgresql-and-mysql/reference-content/security-and-reliability.mdx

```diff
@@ -0,0 +1,223 @@
+
+You can [modify some advanced settings](/managed-databases/postgresql-and-mysql/how-to/configure-advanced-settings/), such as `effective_cache_size` or `max_connections` in PostgreSQL, but you are responsible for any impacts on your database availability and performance.
+
+### Version management
+
+Two types of engine version upgrades are available for your databases:
+
+| Upgrade Type | Description|
+| ------------ | ------------------- |
+| Minor versions | These are updates within the same version and are performed automatically. The updates include security fixes and bug fixes and are designed to be backward compatible. |
+| Major versions    | You can trigger major version upgrades on your Database Instance. Once you activate upgrade, Scaleway automatically creates a clone of your database with the chosen PostgreSQL version. Depending on the option you choose, you might need to manually switch traffic to your newly created database and/or stop write operations on your source database. Refer to the [How to upgrade your Database Instance engine version](/managed-databases/postgresql-and-mysql/how-to/upgrade-version/) documentation page for more information|
+
+### Database extensions
+
+For security reasons, we support a defined set of [databases extensions for PostgreSQL](/faq/databases-for-postgresql-and-mysql/#which-postgresql-extensions-are-available).
+
+For security and reliability reasons, we only support a specific version of each extension for each PostgreSQL major version.
+
+This means we will only update the versions of an extension that are available with our supported PostgreSQL major versions.
+
+As extensions are usually less trusted components, we put in place several security measures to prevent privilege escalations from an extension.
+
+However, the extensions are provided "as is". When installing and using an extension, you remain responsible for its usage, compatibility, and potential side effects on database objects or existing users and queries.
+
+## Data protection
+
+Data protection allows you to safeguard sensitive information from unauthorized access to prevent breaches and data loss. Implementing robust encryption, access controls, and backup strategies ensures data integrity and compliance with regulatory standards.
+
+### Encryption in transit
+Managed Databases for PostgreSQL and MySQL supports SSL connections and allows you to download TLS certificates from Managed Databases.
+
+You remain responsible for configuring encryption for the SQL clients connecting to the database, by carrying out actions such as activating SSL in client configuration options or renewing TLS certificates.
+
+When used by SQL Clients, SSL connections will be performed end-to-end, i.e. from your SQL client up to the Database Instance executing SQL queries.
+
+<Message type="note">
+  Scaleway's components in between, such as load balancers, do not decrypt Managed Databases' traffic.
```

Comment:
```suggestion
  Scaleway's components in between, such as Load Balancers, do not decrypt Managed Databases' traffic.
```

Reply:
Shouldn't it be "Scaleway's in between components"? not sure about this but "Scaleway's components in between" sounds weird to me

Reply:
I dont think so bc in this case it's talking about the specific occurrence and not characterizing these components as always in between, if that makes sense? but we can ask a native

Reply:
```suggestion
  Any intermediary Scaleway components, such as Load Balancers, do not decrypt Managed Databases' traffic.
```
thanks @RoRoJ